app/templates/content/cookies.html

```diff
@@ -5,119 +5,143 @@
 {% endblock %}
 
 {% block breadcrumb %}
-{{ govukBreadcrumbs({
-  "items": [
-    {
+  {{ govukBreadcrumbs({
+      "items": [
+      {
       "href": url_for('main.index'),
       "text": "Digital Marketplace"
-    },
-    {
+      },
+      {
       "text": "Cookies"
-    },
-  ]
-}) }}
+      },
+      ]
+      }) }}
```

Comment:
I don't like to comment on formatting etc., but I'm not convinced this is a helpful change.